lambda/app/src/controllers/logs.ts

```diff
@@ -43,8 +44,8 @@ export const fetchLogs = async (session: Session, env: string, id: number, start
   }
 
   const eventMeta = {
-    requestId: userRequest.id,
-    idirUserid: session.idir_userid,
+    requestId,
+    idirUserid: userId,
```

Comment:
what happens if no userId is provided?

Reply:
It will be null for service accounts, since they don't have one. Downloads through the app UI will include the logged in users id